chat-widget/src/contexts/createReducer.ts

```diff
@@ -326,6 +336,16 @@ export const createReducer = () => {
                     }
                 };
 
+            case LiveChatWidgetActionType.SET_POST_CHAT_WORKFLOW_IN_PROGRESS:
```

Comment:
are we documenting this action types and its purpose?

Reply:
I am currently unsure of any place where we maintain this documentation

Reply:
But i think it might be a good idea to have a documentation of all context variables after the release